security/security.md

```diff
@@ -127,17 +131,26 @@ The default value for the setting is `''`, which means there are no restrictions
+
 #### Redis
 Redis is a lightweight, high-performance key-value database server commonly used to cache the results from WordPress database queries. In its default configuration, Redis uses a single database and does not require a username and password to access the database. Redis should also only be accessible from authorized network hosts.
+
 ##### Redis databases
 Redis provides 16 databases, number 0 to 15 by default. Redis clients should be configured to use different databases instead of the default database (number 0). Redis can be configured to have additional databases, but that it outside the scope of this document.
+
 ##### Redis user credentials
 If Redis is going to be used for database object caching, the Redis server should be configured to require access credentials.
+
 ##### Redis network hosts
 The Redis server in its default configuration listens on port 6379. The port can be changed in Redis's configuration, but whatever port is used should be protected by a firewall to prevent unauthorized access.
+
 ##### Redis cache key salt
 If using Redis for database object caching, using a unique Redis cache key salt will help prevent cache collisions -- when two websites try to cache content using the same key. Cache collisions can result in websites accessing the cached data for other websites and can cause other undesirable and unexpected behaviors. The Redis cache key salt is usually configured through the Redis caching plugin or Redis client used to enable Redis database object caching in WordPress websites.
+
 #### Memcached
-Memcached is a memory object caching solution commonly used to provide database object caching for WordPress. 
+Memcached is a memory object caching solution commonly used to provide database object caching for WordPress.
+
 ##### Memcrashed
-One of the most important configuration concerns for memcached is preventing memcached from being accessed by the public internet. In 2018, insecure memcached servers were used in widespread DDoS amplification attacks using an exploit dubbed Memcrashed. The exploit used memcached servers with open access from the public internet to send large amounts of traffic to DDoS targets in response to spoofed requests. Putting memcached servers behind firewalls is one of the most important parts of using memcached securely for WordPress database object caching.
\ No newline at end of file
+One of the most important configuration concerns for memcached is preventing memcached from being accessed by the public internet. In 2018, insecure memcached servers were used in widespread DDoS amplification attacks using an exploit dubbed Memcrashed. The exploit used memcached servers with open access from the public internet to send large amounts of traffic to DDoS targets in response to spoofed requests. Putting memcached servers behind firewalls is one of the most important parts of using memcached securely for WordPress database object caching.
```

Comment:
I'm not sure we really need the backstory here (do we have a lot of stories like this?) but I suppose it's short enough that it doesn't hurt that much. Just throwing it out there.